app/controllers/users_controller.rb

```diff
@@ -14,6 +14,19 @@ def show
     @repositories = @user.repositories.page(params[:page]).per(PER_PAGE)
   end
 
+  def update_org
```

Comment:
While the link for this action is hidden for users outside the org, this request can be executed for such users with this code.
You should check whether requesting user is in the org or not, reject invalid users and return status code 4XX (bad request).